Formula/b/blst.rb

```diff
@@ -0,0 +1,36 @@
+class Blst < Formula
+  desc "Multilingual BLS12-381 signature library"
+  homepage "https://github.com/supranational/blst"
+  url "https://github.com/supranational/blst/archive/refs/tags/v0.3.11.tar.gz"
+  sha256 "d0a6e2a69490cc45f0a531a684a225e56fe22303665157cfa397ba5605447eb9"
+  license "Apache-2.0"
+
+  def install
+    system "./build.sh", "shared"
+    lib.install "libblst.a"
```

Comment:
Why are we building the shared library but not installing it?

Reply:
mostly because there is no installation step?

Reply:
Then why build the shared library at all?

Reply:
`shared` may be a no-op here, as the script appears to expect `-shared` with a dash: https://github.com/supranational/blst/blob/v0.3.11/build.sh#L43